students/ramkumar_rajanbabu/lesson_01/activity/test_unit.py

```diff
@@ -0,0 +1,174 @@
+"""
+Module Unit Tests
+"""
+
```

Comment:
Instead of breaking the functionality of _main.py_, add the the _inventory_management_ folder to the Python Path by adding this code:

```
import sys
sys.path.append('./inventory_management')
```
With this, you can restore working imports on _main.py_ and remove the import error disabling comment.